app-admin/etcd-wrapper/files/etcd-member.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=etcd (System Application Container)
+Documentation=https://github.com/coreos/etcd
+Conflicts=etcd.service
+Conflicts=etcd2.service
+
+[Service]
+Type=notify
+Restart=always
+RestartSec=10s
+LimitNOFILE=40000
+TimeoutStartSec=60
+Environment=ETCD_NAME=%m
+Environment=ETCD_IMG_USER=etcd
+Environment=ETCD_DATA_DIR=/var/lib/etcd
+Environment=ETCD_TAG=v2.3.7
```

Comment:
It's a tag rather than a version. I _can_ include a version but it does not have to follow the Gentoo versioning scheme. I suggest to keep it simple and bump the revision when the TAG is changed.